cmd/troubleshoot/builder.go

```diff
@@ -105,14 +105,14 @@ func RunTroubleshootCmd(ctx context.Context, log logd.Logger, namespaceName stri
 		return
 	}
 
-	dynakubes := &dynakube.DynaKubeList{}
+	dks, err := getDynakubes(ctx, log, apiReader, namespaceName, dynakubeFlagValue)
+	if err != nil {
+		logErrorf(log, "error during getting dynakues: %v", err)
 
-	err = apiReader.List(ctx, dynakubes, &client.ListOptions{Namespace: namespaceName})
-	if checkCRD(log, err) != nil {
```

Comment:
`checkCRD` is no longer used and can be removed

Reply:
Or even better, use it again, as currently there is no check for the CRD anymore => output is
```
[namespace ] 	 ✓  using namespace 'dynatrace'
[          ] 	no Dynakube specified - checking all Dynakubes in namespace 'dynatrace'
[          ] 	 X  failed to list Dynakubes: no matches for kind "DynaKube" in version "dynatrace.com/v1beta3"
[          ] 	 X  error during getting dynakues: no matches for kind "DynaKube" in version "dynatrace.com/v1beta3"
```

(actually the typo in the last line is really in the code)

Reply:
@gkrenn could you please clarify your comment, it's unclear to me...

Reply:
Sure - What I mean is that with this PR you are removing the function call to `checkCRD`, it's only used in tests anymore. 

This means we don't have a specific check for CRDs anymore in the troubleshoot command. Still due to our errorhandling we produce the "no matches for kind "DynaKube" in version" error. I would rather add the check for the CRD back in again and extend it to other CRDs we are creating. + There is a typo: "dynakues" instead of "dynakubes"

Reply:
I'm okey to return back `checkCRD`, but I don't get the point with extending, the whole file and CLI args are hardcoded to dynakube CR, potentially we can improve it in another ticket if you want.

Reply:
fixed in 619aed90

Reply:
Fine for me